.github/workflows/ci.yml

```diff
@@ -685,3 +685,19 @@ jobs:
           cargo install cargo-check-external-types --locked --version 0.1.6
           cargo check-external-types --all-features --config external-types.toml
         working-directory: tokio
+
+  check-fuzzing:
+    name: check-fuzzing
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust ${{ env.rust_nightly }}
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ env.rust_nightly }}
```

Comment:
Does this require nightly?

Reply:
I confirmed it fails in a commit using `rust_stable` also, but then I `git reset` it to not have it included in the PR.